### PR TITLE
Added Executive Bio to member profile

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**

https://user-images.githubusercontent.com/85789734/175790580-29a3ead4-79c2-4bb6-8b1c-b7ec8f1d0f7a.mov

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@debo19 the recorded demo looks great. If the activated modal can be referenced by URL, I will put the URL to use. Can the photo in the modal be a different, second photo?

**Comment by debo19** [comment]
@leecalcote Yes, the photo can be different. About the url, I'll try to figure it out.

File: src/collections/members/lee-calcote/index.mdx [hunk 1]
```diff
@@ -19,4 +19,13 @@ badges:
 location: Austin, Texas, USA
 bio: "Lee Calcote is an innovative product and technology leader, passionate about empowering engineers and enabling organizations. As Founder of Layer5, he is at the forefront of the cloud native movement. Open source, advanced and emerging technologies have been a consistent focus through Calcote’s time at SolarWinds, Seagate, Cisco and Schneider Electric. An advisor, author, and speaker, Calcote is active in the community as a Docker Captain, Cloud Native Ambassador and GSoC, GSoD, and LFX Mentor."
 published: true
+executive_bio: available
+executive_position: Founder and CEO
+company: Layer5
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/collections/members/lee-calcote/index.mdx:24]: I have added this.

[the remark this answers is not in the document]
Comment (leecalcote) [reply src/collections/members/lee-calcote/index.mdx:24]: Please use this second photo in the alternative bio

**Comment by debo19** [comment]
@Nikhil-Ladha @warunicorn19 do you know why the build is failing?

**Comment by Nikhil-Ladha** [comment]
Is it getting built locally?

File: src/sections/app.style.js [hunk 2]
```diff
@@ -173,8 +176,28 @@ section{
...
+            vertical-align: middle;
+        }
+        .gatsby-image-wrapper, .old-gatsby-image-wrapper {
+            display: block;
+            margin: 1rem auto;
+            border-radius: 100%;
+            overflow: hidden;
+            width: 140px;
+            height: 140px;
+            box-shadow: 0px 0px 8px 1px rgba(0,179,159,1);
```
Comment (warunicorn19) [inline src/sections/app.style.js:195]: Could you please not hard code the color here.

File: src/sections/Community/Member-single/memberSingle.style.js [hunk 3]
```diff
@@ -131,6 +134,10 @@ export const MemberSingleWrapper = styled.div`
                         .twitter{
                             color: #1DA1F2;
                         }
+                        .bio{
+                            color: #00b39f;
```
Comment (warunicorn19) [inline src/sections/Community/Member-single/memberSingle.style.js:138]: Please use a color from the theme.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by debo19** [comment]
> Is it getting built locally?

Yup it does

**Comment by Nikhil-Ladha** [comment]
> > Is it getting built locally?
> 
> Yup it does

I am asking about `make build` not `make site`.

**Comment by debo19** [comment]
> I am asking about `make build` not `make site`.
Yeah both of them are working fine locally.

**Comment by Nikhil-Ladha** [comment]
> > I am asking about `make build` not `make site`.
> 
> Yeah both of them are working fine locally.

Ok
In that case, I think the `react-modal` library is making the bundle heavy and leading to a memory crash. Can you try making a commit with the `Modal` component commented out, and see if that allows the build process or not?

**Comment by leecalcote** [comment]
If the individually published page is too much of a pain in the rump, we can forego it for now, until the exec team is larger.

**Pull request merged**